src/App.jsx

```diff
@@ -14,33 +15,79 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const copyOfGoods = [...goodsFromServer];
+  const [value, setState] = useState('');
+  const [reserve, setReserve] = useState(false);
```

Comment:
Here it is better to rename the states `value` to `sortType` and `reserve` to `isReversed` to improve understanding of their purpose.
```suggestion
  сonst [sortType, setSortType] = useState('');
  const [isReversed, setIsReversed] = useState(false);
```